### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Jst-Tan** [electerious/Ackee#370] [General]
Hi! I want to ask, is it possible to not collect any personal data, including IP address while using the software. 

I don’t want to collect any personal data, since I don’t want to be subjected to any privacy laws, such as GDPR, which say that any data that could be identified to a single individuals.

**Comment by electerious** [comment]
Not collecting personal data is the whole purpose of Ackee. Why do you think it does?

Here's how it handles anonymization in detail: https://github.com/electerious/Ackee/blob/master/docs/Anonymization.md

Ackee just uses the IP to create a hash. It doesn't store the IP nor is it possible to get the IP out of the hash.

> Ackee uses the IP, user-agent and domainId to identify a user. All information will be hashed together with a [salt](https://en.wikipedia.org/wiki/Salt_(cryptography)) that changes daily. The final hash is called clientId.
